### ISSUE TITLE
Sections and win_perf_counters

### ISSUE THREAD

**Issue by maximedevalland** [voxpupuli/puppet-telegraf#68]
I have a question regarding sections and win_perf_counters. I have browse issues and commits of this repo and can't find my answer.

How do you have multiple sections in the same telegraf::input ?

Here's an example : 
```
  telegraf::input { 'memory':
    plugin_type => 'win_perf_counters',
    sections     => {
        'win_perf_counters.object' => {
          'ObjectName'    => 'Memory',
          'Counters' => ["Available Bytes", "Cache Faults/sec", "Demand Zero Faults/sec", "Page Faults/sec", "Pages/sec", "Transition Faults/sec", "Pool Nonpaged Bytes", "Pool Paged Bytes"],
          'Instances' => ["*"],
          'Measurement' => "win_mem",
          }
        }
   }
```

Let's say I want to add Network counters to this. How do you do it ? Because it's only accepting hashes and you can't have the same key twice.

Network counters : 
```
        'win_perf_counters.object' => {
          'ObjectName'    => 'Network Interface',
          'Counters' => ["Bytes Received/sec","Bytes Sent/sec","Bytes Total/sec","Packets Received/sec","Packets Sent/sec"],
          'Instances' => ["*"],
          'Measurement' => "win_net",
        }
```

Thanks!

**Comment by allangood** [comment]
Good point, I'm running into this same problem.

**Comment by yankcrime** [comment]
Fixed via #80.
